src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,135 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exceptions.InvalidRegistrationDataException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private User defaultUser;
+    private StorageDao storageDao;
+    private RegistrationService registrationService;
+
+    @BeforeEach
+    void setUp() {
+        storageDao = new StorageDaoImpl();
```

Comment:
how is this `storageDao` instance connected to the one that you have in the service? do we need this one?
What we do need actually is to not have a unique login for each test with actually clearing the DB `afterEach` method with `Storage.people.clear()`, then we can use the constant login to remove the unique ones for each test because they should  be independent